ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -118,6 +126,79 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: openshift-e2e-vsphere-ccm-ovn-hybrid-install
+- always_run: false
+  as: vsphere-e2e-upgrade
+  steps:
+    cluster_profile: vsphere
+    test:
+    - as: install
+      cli: latest
+      commands: |
+        export WMCO_NS=openshift-windows-machine-config-operator
+        oc create namespace $WMCO_NS
+        oc label --overwrite=true ns $WMCO_NS openshift.io/cluster-monitoring=true \
+        pod-security.kubernetes.io/enforce=privileged
+        # Deploy the bundle which will fail to come up due to it using the operator image via a tag that is not able to
+        # be made public. The csv needs to be patched to point the image via the tag that is pullable by the job.
+        operator-sdk run bundle --timeout=10m --security-context-config restricted -n $WMCO_NS "$PREVIOUS_BUNDLE" \
+        || oc get csv -n $WMCO_NS |awk {'print $1'} | tail -n1 | xargs oc patch csv -n $WMCO_NS --type='json' \
+        -p="[{\"op\": \"replace\", \"path\": \"/spec/install/spec/deployments/0/spec/template/spec/containers/0/image\", \"value\": \"$PREVIOUS_OPERATOR\"}]"
+        sleep 10
+        # Delete the deployment which will then be recreated by the subscription controller with the correct image
+        oc delete deployment -n $WMCO_NS windows-machine-config-operator
+        sleep 5m
```

Comment:
Why do you need the sleep given the wait below?

Reply:
Good question. I dont remember and don't think its actually required. Removing and hoping it works in the pj-rehearse